server/routes/prisons/exclusionDates/exclusionDatesController.test.ts

```diff
@@ -0,0 +1,126 @@
+import type { Express } from 'express'
+import request from 'supertest'
+import * as cheerio from 'cheerio'
+import { FieldValidationError } from 'express-validator'
+
+import { appWithAllRoutes, flashProvider } from '../../testutils/appSetup'
+import { createMockPrisonService, createMockSessionTemplateService } from '../../../services/testutils/mocks'
+import TestData from '../../testutils/testData'
+import { FlashErrorMessage } from '../../../@types/visits-admin'
+
+let app: Express
+let flashData: Record<string, string | FlashErrorMessage>
+
+const prisonService = createMockPrisonService()
+const sessionTemplateService = createMockSessionTemplateService()
```

Comment:
No need to mock this as I don't think this controller uses the `SessionTemplateService`?